### PR TITLE
185602877 enrollment summary

### PR DESCRIPTION
**Pull request by ZachCMP**
https://www.pivotaltracker.com/story/show/185602877

Relies on backend PR: https://github.com/greenriver/hmis-warehouse/pull/3319

### CODE REVIEW

File: src/modules/enrollment/components/EnrollmentDetails.tsx [hunk 1]
```diff
@@ -40,6 +42,7 @@ const EnrollmentDetails = ({
     variables: { id: enrollment.id },
   });
 
+  const enrollmentWithDetails = useMemo(() => data?.enrollment, [data]);
```
Comment (gigxz) [inline src/modules/enrollment/components/EnrollmentDetails.tsx]: This is duplicative of `enrollmentWithCustomElements`, maybe a merge issue

File: src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  const closeDialog = useCallback(() => setDialogOpen(() => false), []);
+  const openDialog = useCallback(() => setDialogOpen(() => true), []);
+
+  const renderDialog = useCallback(
+    (props: Partial<DialogProps> = {}) => {
+      if (!enrollmentSummary) return null;
+
+      return (
+        <CommonDialog {...props} open={!!dialogOpen} onClose={closeDialog}>
```
Comment (gigxz) [inline src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx]: can we give this a fullWidth and maxWidth=md? Looks a little cramped.
Screen Shot 2023-07-27 at 4 27 57 PM

File: src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        ProjectType.Oph,
+        ProjectType.Th,
+        ProjectType.Psh,
+      ];
+
+      if (projectTypes.includes(e.projectType))
+        return parseAndFormatDate(e.moveInDate);
+
+      return null;
+    },
```
Comment (gigxz) [inline src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx]: Lets simplify this to just render the move-in date. The limitation to PH projects is default for collection, but lets just show it if we have it.
```suggestion
    render: (e) => parseAndFormatDate(e.moveInDate)
```

**Comment by ZachCMP** [comment]
@gigxz Addressed your feedback

**Comment by gigxz** [comment]
@ZachCMP have you run `yarn install` lately? it looks like you're using an older version of codegen which is causing conflicts

**Review by gigxz — APPROVED** [review]

**Pull request merged**